src/strategy-arrays.adb

```diff
@@ -39,16 +39,17 @@ package body Strategy.Arrays is
 
          overriding function Simplify (This : in out Node) return Boolean is
          begin
-            while This.Shrinker < Index_Type'Last loop
+            loop
```

Comment:
Why switch to and `loop exit` from the `while loop`?

Reply:
because in the `while loop`, we never get the chance to process `Index_Type'Last`